.strict-typing

```diff
@@ -2,7 +2,7 @@
 # If component is fully covered with type annotations, please add it here
 # to enable strict mypy checks.
 
-# Stict typing is enabled by default for core files.
+# Strict typing is enabled by default for core files.
```

Comment:
This shouldn't be in this PR.

Reply:
Sure, moved to https://github.com/home-assistant/core/pull/79584